File: DashboardDemo/DashboardSampleVB/Dashboard/ColumnHeaderElement.vb
Imports Telerik.WinControls.UI

Public Class ColumnHeaderElement
    Inherits LightVisualElement

    Protected Overrides Sub InitializeFields()
        MyBase.InitializeFields()
        Me.DrawFill = True
        Me.DrawBorder = True
        Me.Padding = New Padding(5)
        Me.StretchHorizontally = True
        Me.StretchVertically = False
        Me.Margin = New Padding(1)
    End Sub
End Class

File: DashboardDemo/DashboardSample/Dashboard/ColumnHeaderElement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Telerik.WinControls.UI;
using System.Windows.Forms;

namespace TicketTest
{
    public class ColumnHeaderElement: LightVisualElement
    {        
        protected override void InitializeFields()
        {
            base.InitializeFields();

            this.DrawFill = true;
            this.DrawBorder = true;
            this.Padding = new Padding(5);
            this.StretchHorizontally = true;
            this.StretchVertically = false;
            this.Margin = new Padding(1);
        }
    }
}
